doc/article/en-US/aurelia-store-plugin.md

```diff
@@ -1645,6 +1645,27 @@ If you've ever worked with Redux then you know for sure about the [Redux Devtool
 
 There are tons of [great articles](https://codeburst.io/redux-devtools-for-dummies-74566c597d7) to get you started. Head over to [DevTools browser extension page](https://github.com/zalmoxisus/redux-devtools-extension) for instructions on how to install the extension, start your Aurelia Store plugin project and see how it works.
 
+## Defining custom devToolOptions
+
+if you use the Redux DevTools extension you can pass options to Aurelia-Store to setup the extension with your preferred configuration.

```

Comment:
since there are multiple configuration possibilities we should add a link for `preferred configuration` to the https://github.com/zalmoxisus/redux-devtools-extension/blob/master/docs/API/Arguments.md